addons/binding/org.openhab.binding.feed.test/pom.xml

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.openhab.binding</groupId>
+		<artifactId>pom</artifactId>
+		<version>2.0.0-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>org.openhab.binding.feed.test</artifactId>
+
+	<name>Feed Binding Tests</name>
+	<packaging>eclipse-test-plugin</packaging>
+
+	<properties>
+		<maven.compiler.version>3.1</maven.compiler.version>
+		<groovy.eclipse.compiler.version>2.8.0-01</groovy.eclipse.compiler.version>
+		<groovy.eclipse.batch.version>2.1.5-03</groovy.eclipse.batch.version>
+		<build.helper.maven.plugin.version>1.8</build.helper.maven.plugin.version>
+	</properties>
+	<repositories>
```

Comment:
When I remove them , I receive error:

> [ERROR] Cannot resolve project dependencies:
[ERROR]   You requested to install 'bundle org.eclipse.equinox.http.jetty 0.0.0' but it could not be found
[ERROR]